docs/payid-discovery.md

```diff
@@ -0,0 +1,46 @@
+---
+id: payid-discovery
+title: Discover a PayID
+sidebar_label: Discover a PayID
+---
+
+The PayID Discovery endpoint facilitates finding the PayID URI for a given PayID.
+
+## PayID discovery by a wallet application
+
+Suppose Alice wants to send Bob some XRP from a web-based wallet provider that Alice has an account on. Alice logs in to the wallet provider and enter Bob's PayID, such as `bob$receiver.example.com`, into the wallet UI to start the payment. If the wallet application is set up to use the PayID Discovery endpoint, the wallet application performs a WebFinger query that looks for the PayID Discovery service provider, like this:
```

Comment:
I really like this narrative approach to introducing the new feature. To make it a touch easier to grok, perhaps give a fictitious name to the " web-based wallet provider"? (At some point, it would be really cool to use actual names and workflows! We'd likely need approval for that, though.)

Reply:
I agree that's a good idea, but I feel compelled to keep example.com as the host until we get clearance otherwise. Thus, a name like WalletXProvider does not work well with example.com. However, I would like for us to have some actual example domains, so will suggest this in the appropriate channels.